core/src/main/java/bio/terra/pearl/core/model/survey/Answer.java

```diff
@@ -31,6 +31,7 @@ public class Answer extends BaseEntity {
     private int surveyVersion;
     private String viewedLanguage;
     private AnswerType answerType;
+    private AnswerFormat format;
```

Comment:
After much going back and forth, I think I've finally settled into being comfortable adding participant files as regular survey response answers with the caveat that we have a new "format" column that dictates how to interpret the data that's present (separate from answer type, which tells you the literal type, e.g. string)